did/document_test.go

```diff
@@ -702,3 +704,36 @@ func TestDocument_IsController(t *testing.T) {
 		assert.False(t, Document{Controller: []DID{*id456}}.IsController(*id123))
 	})
 }
+
+func TestNewVerificationMethod_Multibase(t *testing.T) {
+	// Prepare test data
+	id := DID("did:example:123")
```

Comment:
go complains that I can't use untyped string as DID. I should probably write a String behaviour. Or would that be wrong?

Reply:
`DID` is a struct, not an alias for `string`. Use `ParseDID()` or `MustParseDID()` for parsing a DID.

Reply:
:-)